src/gpodder/gtkui/desktopfile.py

```diff
@@ -169,7 +169,6 @@ def read(self):
 
         self.__has_read = True
         if gpodder.ui.win32:
-            import winreg
```

Comment:
if winreg is not there it would raise the exception. Otherwise it would be logged in the try-catch line 174

Reply:
(ok winreg is in the stdlib so it should be there!)